### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by alissonperes**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,10 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '~>2.7.2'
+ruby '~>3.0.0'
```
Comment (alissonperes) [inline Gemfile:4]: `.ruby-version` file is set to ruby 3.0.0

File: Gemfile [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 
 gem 'rails', '~> 6.1.3', '>= 6.1.3.1'
 
-gem 'pg'
```
Comment (alissonperes) [inline Gemfile:8]: `config/database.yml` isn't using PostgreSQL
This gem isn't needed

File: README.md [hunk 3]
```diff
@@ -42,9 +42,9 @@ Do not use this tool for evil purposes, you hot bum!
 
 ### Prerequisites
 
-- Ruby
-- Bundler
-- Ruby on Rails
+- Ruby (~> 3.0.0)
```
Comment (alissonperes) [inline README.md:45]: Added the ruby version in the README file so it's clear for other people

File: README.md [hunk 4]
```diff
@@ -42,9 +42,9 @@ Do not use this tool for evil purposes, you hot bum!
...
+- Bundler 
+- Ruby on Rails (~> 6.1.3)
```
Comment (alissonperes) [inline README.md:47]: Added the rails version in the README file so it's clear for other people

File: README.md [hunk 5]
```diff
@@ -55,20 +55,22 @@ Do not use this tool for evil purposes, you hot bum!
...
+2. Go to the newly created directory/folder:
 - `cd git-unfollower`
 
-To install the gems from the Gemfile.lock:
+3. To install the gems from the Gemfile.lock:
 - `bundle install`
 
-To start the server locally (http://127.0.0.1:3000/)
+4. To start the server locally (http://127.0.0.1:3000/)
 - `rails s`
```
Comment (alissonperes) [inline README.md]: Changed to an ordered list so it's clear these steps have to be in order.

File: README.md [hunk 6]
```diff
@@ -55,20 +55,22 @@ Do not use this tool for evil purposes, you hot bum!
...
 
+If for some reason you have port 3000 in use, run `rails s -p {PORT_NUMBER}` and access the URL with the desired port.
```
Comment (alissonperes) [inline README.md:75]: This is just nice to have, not really necessary

**Comment by alissonperes** [comment]
@fernandorpm please review my comments and let me know what you think.

**Comment by fernandorpm** [comment]
These changes seem great! Thank you so much for contributing, @alissonperes  :rocket:

**Pull request merged**